index.bs

```diff
@@ -835,11 +846,14 @@ The <dfn>power preference</dfn> indicates preference as related to power consump
     The <dfn method for=ML>createContext(|options|)</dfn> steps are:
 </summary>
 <div class=algorithm-steps>
-    1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, return [=a new promise=] [=rejected=] with a "{{SecurityError}}" {{DOMException}}.
+    1. Let |global| be [=this=]'s [=relevant global object=].
+    1. If |global|'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, return [=a new promise=] [=rejected=] with a "{{SecurityError}}" {{DOMException}}.
+    1. Let |realm| be [=this=]'s [=relevant realm=].
     1. Let |promise| be [=a new promise=].
```

Comment:
optional cleanup here and below: declare `|promise|` before checking the webnn-feature, as to not repeat creating `[=a new promise=]` (could also defer this to a follow-up, if you'd like)

Reply:
I'll defer, and FWIW I prefer the current structure where `|promise|` is only used for the "in parallel" steps, and the almost-synchronous failures are handled in a single line of the spec.

It also doesn't save text:

* If ... then return a new promise rejected with a ....
* If ... then reject promise with a ... and return promise.

Reply:
SGTM 👍